task/control-plane-agent/src/mgs_gimlet.rs

```diff
@@ -960,6 +1097,29 @@ fn configure_usart() -> Usart {
                 }
             };
 
+            // From thin air, pluck a pointer to the USART register block.
+            //
+            // Safety: this is needlessly unsafe in the API. The USART is
+            // essentially a static, and we access it through a & reference so
+            // aliasing is not a concern. Were it literally a static, we could
+            // just reference it.
+            let usart = unsafe { &*device::USART1::ptr() };
+            let peripheral = Peripheral::Usart1;
+            let pins = PINS;
+        } else if #[cfg(feature = "usart1-gimletlet")] {
```

Comment:
This conditional should probably begin with a check that no one has enabled both `usart` and `usart1-gimletlet`, since they're mutually exclusive.

Reply:
Added in a35001ca